### ISSUE TITLE
Why SELECT into new table instead of ALTER TABLE RENAME?

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/kingfisher-views#81]
https://www.postgresql.org/docs/current/sql-altertable.html

We presently have, for example:

```sql
create table planning_documents_summary
AS
select * from staged_planning_documents_summary;

drop table if exists staged_planning_documents_summary;
```

I'm pretty sure that selecting into a new table is much slower than renaming a table.

**Comment by kindly** [comment]
Initially this was the way I wanted to do it.  
The reason for not was due to the indexes on the tables.  Sadly postgres has a global naming of indexes per schema (not per table).  This means that you also need to create indexes on the new table with a different name and then after the rename of the table,  rename all the indexes.  

This proved very fiddly and was difficult to read and get correct and so I opted for the above  solution even though it is not optimal in terms of table access locks (i.e the user will have to wait longer for table to be unlocked this way).

I am now thinking that we have views per collection, just dropping and recreating again would not be such a problem.
